ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -105,6 +105,18 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: baremetalds-ipi-ovn-ipv4-fips-f60
```

Comment:
Do we need to enable fips when moving baremetalds-ipi-ovn-ipv4-f60 from 4.12__arm64-stable to 4.12__amd64-stable.yaml?

Reply:
we disabled fips on arm because arm doesn't support fips,  I checked it always enables fips for baremetalds in nightly jobs https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml#L819